docs/compute-engine/build_converter.md

```diff
@@ -0,0 +1,39 @@
+
+# Building the converter
+
+To build the pip package with the converter, Bazel needs to know for which
+python version to build the package.
+If you are using a python virtual environment, please run bazel from that
+environment, as Bazel will automatically detect the python version of the
+virtual environment.
+
+Bazel will detect the python version the first time you try to build anything
+that requires python. If you have run Bazel outside of your virtual
+environment and now want to run it in the environment, you have to make Bazel
+re-detect the python verion. This requires a full clean:
+
+```bash
+bazel clean --expunge
+```
+
+A normal `clean` is not enough to trigger the redetection.
+
+TODO: required pip packages: `numpy`, `six`. More??
```

Comment:
@lgeiger  Could you verify which packages are required to build the converter? I saw `numpy` and `six` in the `release_linux.sh` script I think, but I expect that also `setuptools` is needed or something like that.

Reply:
usually numpy and six is enough, but I'd recommend to actually install tensorflow since it will make sure that all required pip dependencies are installed. Fixed in a696499f993f5dae745a98b9d3ccd232340c77f5